webapp/src/components/post_menu_action/create_issue/index.js

```diff
@@ -16,7 +16,7 @@ const mapStateToProps = (state, ownProps) => {
     const systemMessage = post ? isSystemMessage(post) : true;
 
     return {
-        show: state[`plugins-${pluginId}`].connected && !systemMessage,
+        show: state[`plugins-${manifest.id}`].connected && !systemMessage,
```

Comment:
Not required for this PR, but it would be good to always use a selector for this sort of thing, and never do this `plugins-(id)` thing in these files. Can this be done as part of the redux store migration?